utils/wizard.mts

```diff
@@ -1,103 +1,109 @@
-import inquirer from 'inquirer';
-
+import inquirer from "inquirer";
 
 type Answers = {
-    componentName: string,
-    framework: string,
-    template: string,
-    folder: string
-}
-const wizard = async()=>{
-
-    return inquirer.prompt([
-        {
-            type: "input",
-            name: "componentName",
-            message: "give a name to your component",
-            validate: (input) => {
-                const trimmedInput = input.trim();
-                if (trimmedInput === '') {
-                    return 'Component name cannot be empty';
-                }
-                // Use a regular expression to check for only alphanumeric characters
-                const isValid = /^[a-zA-Z0-9]+$/.test(trimmedInput);
-                return isValid ? true : 'Component name can only contain alphanumeric characters';
-            }
+  componentName: string;
+  framework: string;
+  template: string;
+  folder: string;
+};
+const wizard = async () => {
+  return inquirer
+    .prompt([
+      {
+        type: "input",
+        name: "componentName",
+        message: "give a name to your component",
```

Comment:
Since you're working on it, can you fix it here too? Thank you so much

Reply:
For sure!

Reply:
Done!